src/main/java/org/jabref/model/JabRefStringConstants.java

```diff
@@ -0,0 +1,12 @@
+package org.jabref.model;
+
+public class JabRefStringConstants {
```

Comment:
Maybe missing a private empty constructor, so this class can't be instantiated.
Codewise LGTM, although I really don't know if this is good practice. But to move forward an calm @koppor s temper...

Reply:
Agree with @calixtus not sure either. I would at least rename the class to PushToApplicationConstants and would have expected them in a supackge model/push/

Reply:
The main idea behind such a generic class was that there are multiple cases in the JabRefPreferences where GUI components are referenced for accessing names and thus we could collect them in a single file.
Is having multiple small constant string files prefrerrable?

Reply:
Name of the class came up yesterday as we were thinking about collecting all the commonly used constant strings in jabref in that class.

Please add the private constructor, then we should merge, to start with the little things.